src/test/java/com/amazon/ion/impl/IonReaderContinuableCoreBinaryTest.java

```diff
@@ -1,46 +1,38 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-
 package com.amazon.ion.impl;
 
 import com.amazon.ion.IonCursor;
 import com.amazon.ion.IonException;
 import com.amazon.ion.IonType;
+import com.amazon.ion.TestUtils;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
 import org.junit.jupiter.params.provider.ValueSource;
 
 import java.io.ByteArrayInputStream;
 
 import static com.amazon.ion.BitUtils.bytes;
-import static com.amazon.ion.impl.IonCursorTestUtilities.STANDARD_BUFFER_CONFIGURATION;
-import static com.amazon.ion.impl.IonCursorTestUtilities.Expectation;
-import static com.amazon.ion.impl.IonCursorTestUtilities.ExpectationProvider;
-import static com.amazon.ion.impl.IonCursorTestUtilities.assertSequence;
-import static com.amazon.ion.impl.IonCursorTestUtilities.container;
-import static com.amazon.ion.impl.IonCursorTestUtilities.container;
-import static com.amazon.ion.impl.IonCursorTestUtilities.endContainer;
-import static com.amazon.ion.impl.IonCursorTestUtilities.endStream;
-import static com.amazon.ion.impl.IonCursorTestUtilities.fillContainer;
-import static com.amazon.ion.impl.IonCursorTestUtilities.fillIntValue;
-import static com.amazon.ion.impl.IonCursorTestUtilities.scalar;
-import static com.amazon.ion.impl.IonCursorTestUtilities.scalar;
-import static com.amazon.ion.impl.IonCursorTestUtilities.startContainer;
-import static com.amazon.ion.impl.IonCursorTestUtilities.fillStringValue;
+import static com.amazon.ion.impl.IonCursorTestUtilities.*;
```

Comment:
I don't really care what style we choose for imports, but if there is an editor or linter setting that folds imports into wildcards it may cause a lot of extra diff noise in files we touch in ion-java.

Reply:
Sure, I can revert that.